packages/teams-js/src/public/dialog.ts

```diff
@@ -62,6 +65,34 @@ export namespace dialog {
     sendMessageToParent('tasks.completeTask', [result, Array.isArray(appIds) ? appIds : [appIds]]);
   }
 
+  /**
```

Comment:
@erinha Are there documentation guidelines we can use to verify we've got documentation that meets our standards?

Reply:
We have used tsdoc style in past for document comments. https://tsdoc.org/
But it might have changed. I am open Erin's advice! :)

Reply:
Please do follow up with Erin and ask if we need to do anything to get the comments looked at by our documentation people :)